public/app/features/admin/ldap/LdapSettingsPage.tsx

```diff
@@ -0,0 +1,315 @@
+                  />
+                </Field>
+                <Field htmlFor="bind-password" label={t('ldap-settings-page.bind-password.label', 'Bind password')}>
+                  <Input
+                    id="bind-password"
+                    type="text"
+                    {...register('settings.config.servers.0.bind_password', { required: false })}
+                  />
+                </Field>
+                <Field
+                  htmlFor="search_filter-filter"
+                  label={t('ldap-settings-page.search_filter.label', 'Search filter*')}
+                  description={t(
+                    'ldap-settings-page.search_filter.description',
+                    'LDAP search filter used to locate specific entries within the directory.'
+                  )}
+                >
+                  <Input
+                    id="search_filter"
+                    placeholder={t('ldap-settings-page.search_filter.placeholder', 'example: cn=%s')}
+                    type="text"
+                    {...register('settings.config.servers.0.search_filter', { required: true })}
+                  />
+                </Field>
+                <Field
+                  htmlFor="search-base-dns"
+                  label={t('ldap-settings-page.search-base-dns.label', 'Search base DNS *')}
+                  description={t(
+                    'ldap-settings-page.search-base-dns.description',
+                    'An array of base dns to search through; separate by commas or spaces.'
+                  )}
+                >
+                  <Input
+                    id="search-base-dns"
+                    placeholder={t('ldap-settings-page.search-base-dns.placeholder', 'example: "dc=grafana.dc=org"')}
+                    type="text"
+                    {...register('settings.config.servers.0.search_base_dns', { required: true })}
+                  />
+                </Field>
+                <Box display={'flex'} gap={2} marginTop={5}>
+                  <Stack alignItems={'center'} gap={2}>
+                    <Button type={'submit'}>
+                      <Trans i18nKey="ldap-settings-page.buttons-section.save-and-enable.button">Save and enable</Trans>
+                    </Button>
+                    <Button variant="secondary" onClick={saveForm}>
+                      <Trans i18nKey="ldap-settings-page.buttons-section.save.button">Save</Trans>
+                    </Button>
+                    <Button variant="secondary" onClick={discardForm}>
+                      <Trans i18nKey="ldap-settings-page.buttons-section.discard.button">Discard</Trans>
+                    </Button>
+                  </Stack>
+                </Box>
+              </section>
+            )}
+          </form>
+        </FormProvider>
+      </Page.Contents>
+    </Page>
+  );
+};
+
+function getStyles(theme: GrafanaTheme2) {
+  return {
+    form: css({
+      input: css({
+        width: theme.spacing(68),
```

Comment:
You can add max-width to the form instead and the inputs will take up all the available space.